MyJobApplication/src/com/amaderorg/myjobapp/Model/Database/Tables/ContactInformation.java

```diff
@@ -0,0 +1,74 @@
+package com.amaderorg.myjobapp.Model.Database.Tables;
+
+/**
+ * Created by souvi_000 on 3/28/2016.
+ */
+public class ContactInformation {
+    public String getFirstName() {
+        return mFirstName;
+    }
+
+    public void setFirstName(String mFirstName) {
+        this.mFirstName = mFirstName;
+    }
+
+    public String getLastName() {
+        return mLastName;
+    }
+
+    public void setLastName(String mLastName) {
+        this.mLastName = mLastName;
+    }
+
+    public String getCompanyName() {
+        return mCompanyName;
+    }
+
+    public void setCompanyName(String mCompanyName) {
+        this.mCompanyName = mCompanyName;
+    }
+
+    public int getLocation() {
+        return mLocation;
+    }
+
+    public void setLocation(int mLocation) {
+        this.mLocation = mLocation;
+    }
+
+    public String getEmailId() {
+        return mEmailId;
+    }
+
+    public void setEmailId(String mEmailId) {
+        this.mEmailId = mEmailId;
+    }
+
+    private String mFirstName;
```

Comment:
To follow consistency, please define the methods of this class after the constructor.